src/utils/flowRight.ts

```diff
@@ -0,0 +1,16 @@
+export default(...funcs: any[]) => {
+  const functions = funcs.reverse();
+  functions.forEach((fnc, i) => {
+    if (typeof fnc !== 'function') {
+      throw new Error(`In "compose" the argument at index "${i}" is not a function.`);
+    }
+  });
+  return function (...args: any[]) {
+    const [firstFunction, ...restFunctions] = functions
+    let result = firstFunction.apply(null, args);
+    (restFunctions || []).forEach((fnc) => {
```

Comment:
```suggestion
    restFunctions.forEach((fnc) => {
```
I believe `restFunctions` is guaranteed to be an array here by the `...restFunctions` syntax above.